net.christianbeier.Gromit-MPX.yml

```diff
@@ -38,8 +38,8 @@ modules:
       - /share/dbus-1
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/dconf/0.38/dconf-0.38.0.tar.xz
-        sha256: 45f60f41330d27715cce1315af123f94f1c2cdedb68b6bed3b309866eec44f58
+        url: https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
```

Comment:
Did you test that the dconf-backed hotkey setting under Wayland still works?

Reply:
I tested that under Gnome 42, wayland it grabs F8 by default I could toggle the onscreen writing tool with it and start writing. Is that what you want?

Reply:
This is what the users want, yes ;-)

Reply:
:heavy_check_mark: